docs/sql-programming-guide.md

```diff
@@ -1798,8 +1798,7 @@ DataFrame jdbcDF = sqlContext.load("jdbc", options)
 
 {% highlight python %}
 
-df = sqlContext.load(source="jdbc", url="jdbc:postgresql:dbserver", dbtable="schema.tablename")
-
+df = sqlContext.read.format('jdbc').options(url = 'jdbc:postgresql:dbserver', dbtable='schema.tablename').load()
```

Comment:
I may learn something but does this syntax work? it looks like it expects a `Map`. I am just eyeballing the docs though so could be wrong